docker-compose-oidc.yaml

```diff
@@ -1,130 +0,0 @@
-version: "3.6"
-services:
-  play:
-    environment:
-      #OPID_CLIENT_ID: authorization-code-with-pkce-client-id
-      #OPID_CLIENT_SECRET: authorization-code-with-pkce-client-secret
-      OPID_CLIENT_ID: authorization-code-client-id
-      OPID_CLIENT_SECRET: authorization-code-client-secret
-      OPID_CLIENT_ISSUER: http://oidc.workadventure.localhost
-      OPID_SCOPE: profile openid email tags-scope
-      MAP_EDITOR_ALLOW_ALL_USERS: "false"
-
-  # A mock server to test OpenID connect connectivity
-  oidc-server-mock:
```

Comment:
You are deleting the oidc-server-mock container here. That's fine, but you need to add it back to docker-compose.yamlù!